### DISCUSSION TITLE
Is there a timetable when (paid) plugins for Kimai v2 are made available?

### DISCUSSION THREAD

**Discussion by BeckeBauer** [kimai/kimai#3943] [Plugins]
I was wondering when to expect plugins becoming available to be used with Kimai v2 since they can be tested and work in the demo version? 
I would love to upgrade to Kimai v2...

**Comment by kevinpapst** [comment]
I wanted to wait for MR Software, but it seems they need (some) more time.

Really sorry that it takes so long, I am working on it. 
Actually most of it is ready, but I am moving to a new platform for selling and that takes most of the time and preparation.

On the good side: I was able to fix some many tiny issues in the meantime, thanks to the feedback from all cloud users, that you will have less of them.

**Comment by BeckeBauer** [comment]
Thanks for the update... I will patiently wait (one of my few strength 😬)
